### ISSUE TITLE
[Design] Profile - Confirm with VHA & VBA how info should be updated

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#33007]
## Introduction 
In an email conversation (see attached PDF for reference), it was revealed that the process for updating some of the personal information fields has changed on the back end. 

## Tasks
- [x] Confirm with VHA how people should update name/birth sex/birth date
- [ ] Confirm with VBA how people should update name/birth sex/birth date (this has not been started and is for people who do not receive health care)
- [ ] Updating the FAQ addressing this in the profile

## Acceptance Criteria
- [x] Confirm with VHA how people should update name/birth sex/birth date
- [ ] Confirming with VBA how people should update name/birth sex/birth date
- [ ] Updating the FAQ addressing this in the profile

[Email thread for reference](https://app.zenhub.com/files/133843125/a5d0e180-415c-482d-977b-045067e2c91c/download)

**Comment by andaleliz** [comment]
Camille and Nick at VBA stated Veterans should be directed to the call center if they need to change any of the items in question. The Veteran will have to submit documentation for the change. However, I found [this information](https://www.va.gov/resources/how-to-change-your-legal-name-on-file-with-va/#change-your-name-for-va-benefi) in our resources and support section, which suggests Veterans could do the whole process without necessarily calling if they don't want to. 

@Samara-Strauss, let me know if you hear anything from Melissa. I can reach out to Danielle about the content I found and see if she knows who could tell us if the process outlined on the page would also apply for birth date/sex changes. I assume so but would like to validate.

**Comment by Samara-Strauss** [comment]
OMG, this is GREAT and I had no idea this existed! No need for us to reinvent the wheel. If the VHA folks who got in touch say this is good, then we should just link to this from the profile. Thanks for finding this, @andaleliz!

**Comment by andaleliz** [comment]
>If the VHA folks who got in touch say this is good, then we should just link to this from the profile.

That's not exactly right - I found this link after I spoke to them. VHA folks said to direct someone to the call center. This link suggests an alternate approach. I think on our FAQ, we can include both and say something like:

**If you’re enrolled in the VA health care program**
Please contact your nearest VA medical center to update your personal information.

Find your nearest VA medical center

**If you receive VA benefits, but aren’t enrolled in VA health care**
You can call us at 800-827-1000 (TTY: 711). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. Or, [you can request this update by mail](https://www.va.gov/resources/how-to-change-your-legal-name-on-file-with-va/#change-your-name-for-va-benefi).

**Comment by andaleliz** [comment]
Oh, the other thing I'm not sure about is if the name change process on that page applies to changing DOB or birth sex. I imagine so but would love to confirm that with someone.

**Comment by MarciMcGuireGCIO** [comment]
@andaleliz is this the separate update processes being discussed in the email thread Samara just looked Danielle into? It sounds like they are asking the same questions, but perhaps for different purposes.

**Comment by andaleliz** [comment]
@MarciMcGuireGCIO yes - that convo is for VHA. My comments are about VBA :)

**Comment by MarciMcGuireGCIO** [comment]
Oh, thank you! 🤦‍♀️

**Comment by andaleliz** [comment]
Last week, I emailed back and forth with some VBA folks to try to determine how a Veteran who doesn't have health benefits should approach this. It's more complicated than I could've imagined. Frank Bryceland (VBA benefit assistance service) explained that EVSS is likely the best place to start because data is stored in many different places and they'd have the greatest ability to access those places.  If they couldn't make an update in a database, they'd likely be able to advise the Veteran on who to get in touch with.

Frank referred me to an EVSS contractor, Jeremy Haas - here's the last email I got from Jeremy on 11/23:
> I have spoken with Dale and also other members of my team regarding this email.  The EVSS HD has a great deal of experience resolving many data mismatch issues related to veteran records as you and Frank described.  However, the question on what number to provide for veterans to use, is a broader subject, which would require input and agreement from several individuals and teams that are involved in the process. To be sure that any recommendation being provided is accurate and will work effectively, I first need to have those conversations with them.  I will setup a meeting to discuss this with Dale and the others, and I will respond back after ensuring that all parties are in agreement, in a recommendation. 

I'll follow up with Jeremy on 12/3 if I don't hear anything back by then.  Moving into sprint 62.

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @andaleliz.  Nothing can ever be simple, can it?  😆

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. You had mentioned that you were confirming the process for the other info. Could you let me know when you have confirmation and who the stakeholders are that you're working with? Thank you!

**Comment by andaleliz** [comment]
Hi @DanielleThierryUSDSVA! By other info, do you mean VHA? If so, the stakeholder we worked with is named Kimberly Womack with VHA Office of Health Informatics, and she confirmed all roads lead to their VAMC. A Veteran can go in person to their VAMC to make updates, or send a written request.  To know where to send a written request, they'd have to contact their VAMC by phone.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. I was referring to VBA. I'm on an email chain with Kimberly from VHA, but I thought you'd also said you were confirming things on the VBA side too. Is that incorrect?

**Comment by andaleliz** [comment]
@DanielleThierryUSDSVA - that is correct. My comment prior to yours was VBA so the `other` threw me off.  I don't have additional info beyond that at the moment but I'll share an update here when I do. I'm planning on following up with Jeremy Haas this Friday if I haven't heard more from him by then.

**Comment by andaleliz** [comment]
Sent a follow up email to Jeremy for VBA information.

For VHA, I think we can keep as-is.  The page in resources and support is only about the name, so I think linking people there could cause confusion if they instead or also want to update their birth sex or DOB. Danielle thinks there is value in adding info about changing birth sex and DOB to the R&S page, so once that it done, we can link to it. [I've asked her if there are any existing tickets or anything we can follow along with](https://dsva.slack.com/archives/C01K37HRUAH/p1638803982156400?thread_ts=1637615573.120000&cid=C01K37HRUAH) so we know when that update has been made.

An additional improvement could be updating the info in the FAQ based on what we know about someone.  Is there already logic to show the relevant info based on what we know about the benefits they have? Seems like at a minimum we could only show the health care message when we know someone has health care, and update the heading for the 2nd part (about non-health benefits) accordingly.

I believe Samara mentioned moving forward with VHA things while we wait for VBA. Since we're not making changes, I don't think any FE ticket is needed for that part of this work, unless we want to explore the conditional messaging I described above.

**Comment by Samara-Strauss** [comment]
> I've asked her if there are any existing tickets or anything we can follow along with so we know when that update has been made.

Great. Once the info on birth date/birth sex is in the R&S section, we can update the FAQ and link over there.

> An additional improvement could be updating the info in the FAQ based on what we know about someone. Is there already logic to show the relevant info based on what we know about the benefits they have? Seems like at a minimum we could only show the health care message when we know someone has health care, and update the heading for the 2nd part (about non-health benefits) accordingly.

I don't think it's worth the LOE to make this FAQ dynamic, especially when we know we'll eventually link over to the R&S section. It wouldn't make sense to introduce a bunch of logic when we know we'll rip that out fairly soon.

Agree no FE ticket is necessary now, though we may want to create another ticket to update the FAQ once the R&S content for birth date and birth sex is ready.

**Comment by andaleliz** [comment]
@Samara-Strauss that all makes sense, and good call on the LOE.

I went ahead and [mocked-up the page when we can use a link instead of an FAQ](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/zx0GAGa), while it's fresh in my mind. It may require a small copy change depending on the h1 they decide on once they update the R&S page.

**Comment by Samara-Strauss** [comment]
Thank you, Liz!

Since the plan is to link to R&S content when it's ready, but that work is tbd on when it will be done, we can close this ticket and open a new one for FE to implement the new design and link to R&S when that is ready. But that will likely sit in the backlog for some time.

**Comment by MarciMcGuireGCIO** [comment]
I will create a FE ticket for this.

**Comment by andaleliz** [comment]
Documenting the response I got from Jeremy yesterday. Will post more when I hear back from him again.

>Yes I do some update for you on the question of who your team should be referring veterans to call. I have a meeting scheduled this week, involving the stakeholders from the multiple teams that would be involved in a new routing process of who your team can refer these DEERS add/update related cases too. There are still questions and details we need to work through, and also to ensure that there is proper approval and all the stakeholders are onboard. So unfortunately I do not have an recommendation at this time but, I am still working to get one.

**Comment by MarciMcGuireGCIO** [comment]
FE ticket for linking to R&S article once complete:  #33757

**Comment by MarciMcGuireGCIO** [comment]
Blocked; need info for updating VBA

**Comment by MarciMcGuireGCIO** [comment]
R&S article for updating Legal Name is live - https://www.va.gov/resources/how-to-change-your-legal-name-on-file-with-va/

**Comment by andaleliz** [comment]
Thanks Marci! I sent a follow-up to Jeremy for VBA info to make sure that doesn't get too stale.

**Comment by MarciMcGuireGCIO** [comment]
Thank you @***@***.******@***.***>!
________________________________

**Comment by MarciMcGuireGCIO** [comment]
@andaleliz - since it's been a while, should I or Christina reach back out to Jeremy re: VBA info?  Thx!

**Comment by andaleliz** [comment]
Thanks for the reminder! I just sent a follow-up and added Christina to the change so she can take over in the future if needed.  Hopefully they have something for us by now.

**Comment by andaleliz** [comment]
Got an OOO reply from Jeremy that he'll be out until 1/10; will re-send my follow up on 1/11 to bump it to the top of his inbox and give him a day to catch up.

**Comment by MarciMcGuireGCIO** [comment]
Ok, thanks!

**Comment by andaleliz** [comment]
@MarciMcGuireGCIO re-assigning to you per your generous offer to take over follow ups.

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @andaleliz!

**Comment by MarciMcGuireGCIO** [comment]
Followed up with Jeremy Hass via email on 1/19/2022.  Below is his response: 

>Hello. I did meet with Chante and NCC to discuss this last week.  We are waiting for some follow up documentation from NCC, on the fields that they are updating, in order to answer that question for you.  I will update you when I know though. 

>Jeremy

**Comment by MarciMcGuireGCIO** [comment]
Received confirmation from Jeremy today that Veterans need to contact National Call Centers to update their Benefits info
